api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirDiagnosticReportDaoImpl.java

```diff
@@ -45,4 +54,46 @@ public Obs saveObsGroup(Obs obs) throws DAOException {
 		
 		return obs;
 	}
+	
+	@Override
+	public Obs get(String uuid) {
+		// TODO Auto-generated method stub
+		return super.get(uuid);
+	}
+	
+	@Override
+	public Obs createOrUpdate(Obs newEntry) {
+		// TODO Auto-generated method stub
+		return super.createOrUpdate(newEntry);
+	}
+	
+	@Override
+	public Obs delete(String uuid) {
+		// TODO Auto-generated method stub
+		return super.delete(uuid);
+	}
+	
+	@Override
+	public Long getResultCounts(SearchParameterMap theParams) {
+		// TODO Auto-generated method stub
+		return super.getResultCounts(theParams);
+	}
+	
+	@Override
+	public Integer getPreferredPageSize() {
+		// TODO Auto-generated method stub
+		return super.getPreferredPageSize();
+	}
+	
+	@Override
+	public Collection<Obs> search(SearchParameterMap theParams) {
+		// TODO Auto-generated method stub
+		return super.search(theParams);
+	}
+	
+	@Override
+	protected void setupSearchParams(Criteria criteria, SearchParameterMap theParams) {
+		// TODO Auto-generated method stub
+		super.setupSearchParams(criteria, theParams);
+	}
```

Comment:
I wouldn't include these overrides unless we specifically need them. The idea is that the documentation should live in the interface and the implementation in the base class as much as possible.

Reply:
I appreciate all the inputs , i think others will be more alligned with Fhir.